jenkins/scripts/files/run_fetch_logs.sh

```diff
@@ -27,13 +34,13 @@ do
   done
 done
 
-mkdir -p "${LOGS_DIR}/docker"
-DOCKER_CONTAINERS="$(docker ps --format "{{json .}}" | jq -r '.Names')"
-for DOCKER_CONTAINER in $DOCKER_CONTAINERS
+mkdir -p "${LOGS_DIR}/${CONTAINER_RUNTIME}"
+LOCAL_CONTAINERS="$("${CONTAINER_RUNTIME}" ps --format "{{json .}}" | jq -r '.Names')"
+for LOCAL_CONTAINER in $LOCAL_CONTAINERS
 do
-  mkdir -p "${LOGS_DIR}/docker/${DOCKER_CONTAINER}"
-  docker logs "$DOCKER_CONTAINER" > "${LOGS_DIR}/docker/${DOCKER_CONTAINER}/stdout.log" \
-  2> "${LOGS_DIR}/docker/${DOCKER_CONTAINER}/stderr.log"
+  mkdir -p "${LOGS_DIR}/${CONTAINER_RUNTIME}/${LOCAL_CONTAINER}"
+  "${CONTAINER_RUNTIME}" logs "$LOCAL_CONTAINER" > "${LOGS_DIR}/${CONTAINER_RUNTIME}/${LOCAL_CONTAINER}/stdout.log" \
```

Comment:
For some reason there are no logs saved from podman

Reply:
that should be fixed now. Podman didn't like the template string

Reply:
and was missing sudo :/